ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -41,12 +41,13 @@
       COUCHDB_PASSWORD: "{{ db_password }}"
       NODENAME: "{{ nodeName }}"
 
-- name: wait until the CouchDB in this host is up and running
-  wait_for:
-    delay: 2
-    host: "{{ ansible_host }}"
-    port: "{{ db_port }}"
-    timeout: 60
+- name: wait until CouchDB in this host is up and running
```

Comment:
wasn't this part of a different pr?

Reply:
yes it was https://github.com/apache/incubator-openwhisk/commit/602ce9438cdf5cbfa5e3365223c1c48348ff1092.
just need a rebase.